docs/architecture/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-layout: default
-title: Architecture
+layout: i18n_page
+title: pages.architecture
```

Comment:
Related to earlier questions on whether we need to ensure updates to the `en.yml` need to be checked in... for generated pages do we need to generate the english translations for the pages too?

Reply:
Thanks for clarifying on chat, that this phase will add the infrastructure to allow contributions, but the switch to having it on across the board only really makes sense once the first set of contributions is in place. So we will defer that work for now.